twitter/views_admin.py

```diff
@@ -31,10 +31,8 @@ def retrieve_possible_twitter_handles_view(request, candidate_campaign_we_vote_i
 
     candidate_campaign = results['candidate_campaign']
 
-    # TODO Uncomment this and work within here
-    # results = retrieve_possible_twitter_handles(candidate_campaign)
-    number_of_possibilities_found = 0
-    messages.add_message(request, messages.INFO, 'Number of possibilities found: ' + str(number_of_possibilities_found))
+    results = retrieve_possible_twitter_handles(candidate_campaign)
+    messages.add_message(request, messages.INFO, 'Number of possibilities found: ' + results['num_of_possibilities'])
```

Comment:
Thank you Daniel for all of this great work. One request: I try to avoid abbreviations like "num" and prefer to use full words in variables names like "number". 
https://github.com/wevote/WebApp/blob/develop/docs/contributing/CONTRIBUTING_STANDARDS.md